### PR TITLE
Added Python in Language.md

### PR DESCRIPTION
**Pull request by amritsinghcse**

### CODE REVIEW

**Comment by mbad0la** [comment]
@prabhakar267 please review this.

**Comment by prabhakar267** [comment]
@mbad0la I will see it in a day or two

File: Language.md [hunk 1]
```diff
@@ -69,3 +69,27 @@ Technologies like React Native, Cordova/Ionic provide JS tech stack for Mobile A
...
+ 
+**2. Internet of things opportunities.**
+ 
+ Python may become popular for the Internet of things, as new platforms such as Raspberry Pi are based on it, Carbonnelle says. [Raspberry Pi's documentation](https://www.raspberrypi.org/documentation/usage/python/) cites the language as "a wonderful and powerful programming language that's easy to use (easy to read and write) and with Raspberry Pi lets you connect your project to the real world."
+
+**3. Asynchronous coding benefits.**
+
+Python, Deibel says, "is great for writing asynchronous code, which rather than threading uses a single event loop to do work in small units." This code, he says, is often easier to write and maintain without confusing resource contention, deadlocks, etc. "Python's generators are a great way to interleave running many processing loops in this approach." 
+
+**4. Multiparadigm approach bests Java.**
```
Comment (prabhakar267) [inline Language.md]: beats*

**Comment by prabhakar267** [comment]
@mbad0la  Looks fine to me!
@SuperHaker Great work :+1:

**Comment by amritsinghcse** [comment]
@prabhakar267 Thanks. Merge it if rest is fine.

**Pull request merged**